app/services/pagelets/manager.rb

```diff
@@ -0,0 +1,35 @@
+module Pagelets
+  class Manager
+    class << self
+
+      def add_pagelet(page_name, mountpoint, opts)
+        handle_empty_keys_for page_name, mountpoint
+        raise ::Foreman::Exception.new("Cannot add pagelet to page #{page_name} without partial") unless opts[:partial]
+        raise ::Foreman::Exception.new("Cannot add pagelet to page #{page_name} without mountpoint") if mountpoint.nil?
+        priority = opts[:priority] || default_pagelet_priority(page_name, mountpoint) - 100
```

Comment:
Why decreasing? I thought that priority is in ascending order, therefore I'd expect `+ 100`. Is it in descending?